### DISCUSSION TITLE
Please change the default binding (`enter`) for force pushes

### DISCUSSION THREAD

**Discussion by bennypowers** [jesseduffield/lazygit#1905] [Ideas]
Force pushing is a destructive operation. The confirmation dialog which asks whether you'd like to cancel or force push should not use the `enter` key to confirm, but a something unusual, I propose `f`.

This is because the consequences of mistakenly force pushing (e.g. if the user did not carefully read the dialog) could be dire, while the consequences of cancelling are mild.

**Comment by AndrewSav** [comment]
At least we need a way to configure that to our own preference. I've force pushed with lazygit on accident before.

**Reply by mark2185** [reply]
Could you create an issue for that, for easier bookkeeping?

**Reply by AndrewSav** [reply]
@mark2185 [here we go](https://github.com/jesseduffield/lazygit/issues/2009)

**Reply by mark2185** [reply]
Great, thanks, it'll get to the top of the `TODO` stack sooner or later.
